galasa-parent/dev.galasa.framework.api.cps/src/main/java/dev/galasa/framework/api/cps/internal/routes/AddPropertyInNamespaceRoute.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.cps.internal.routes;
+
+import static dev.galasa.framework.api.common.ServletErrorMessage.*;
+
+import java.io.IOException;
+
+import java.nio.charset.StandardCharsets;
+
+import javax.servlet.ServletException;
+import javax.servlet.ServletInputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.JsonObject;
+
+import dev.galasa.framework.api.beans.GalasaProperty;
+import dev.galasa.framework.api.common.InternalServletException;
+import dev.galasa.framework.api.common.QueryParameters;
+import dev.galasa.framework.api.common.ResponseBuilder;
+import dev.galasa.framework.api.common.ServletError;
+import dev.galasa.framework.api.common.resources.CPSFacade;
+import dev.galasa.framework.api.common.resources.CPSNamespace;
+import dev.galasa.framework.api.common.resources.CPSProperty;
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.FrameworkException;
+import dev.galasa.framework.spi.IFramework;
+
+public class AddPropertyInNamespaceRoute extends CPSRoute {
+
+    protected static final String path = "\\/namespace/([a-z][a-z0-9]+)/property/([a-zA-Z0-9\\.\\-\\_]+)/?";
```

Comment:
Do we need to escape the `/` characters? We're escaping the first one, but the rest are as is (Java might actually be fine with it)

Same goes for all the other new routes